package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@teamleader/ui",
   "description": "Teamleader UI library",
-  "version": "18.6.2",
+  "version": "18.6.3",
```

Comment:
if its breaking change then you should go with major update

Reply:
Knocks like a bus! 🚌 

JK, you're right! I will update this ;)